rllib/agents/slateq/slateq_tf_policy.py

```diff
@@ -157,21 +157,18 @@ def build_slateq_losses(
     q_clicked = tf.gather(replay_click_q, clicked_indices)
     target_clicked = tf.gather(target, clicked_indices)
 
-    def get_train_op():
-        td_error = q_clicked - target_clicked
-        if policy.config["use_huber"]:
-            loss = huber_loss(td_error, delta=policy.config["huber_threshold"])
```

Comment:
did we get rid of the option to use these huber losses? Why remove all of this, and replace with only one option for the loss? Just curious, but otherwise lgtm